api/src/main/java/org/openmrs/module/appointments/service/Test.java

```diff
@@ -0,0 +1,29 @@
+package org.openmrs.module.appointments.service;
```

Comment:
@angshu do we need this file ?

Reply:
I accidentally had checked in the file, but in another subsequent commit I deleted the file. Its not there on my history. maybe git diff is not displaying properly?